ansible/roles/docker/tasks/main.yml

```diff
@@ -5,6 +5,9 @@
 - include: apt-docker-install.yml
   when: ansible_distribution == "Ubuntu"
 
+- include: use-docker-repos.yml
+  when: docker_provider == "docker"
```

Comment:
Shouldn't this be gated on ansible_pkg_mgr == yum or dnf? I don't want to run this on a apt/dpkg based system. Update that and I will merge.